### PR TITLE
Fix salaries perms

### PR DESCRIPTION
**Pull request by orebd**
Fix user acces by defaut to all employees salaries

### CODE REVIEW

File: htdocs/langs/fr_FR/admin.lang [hunk 1]
```diff
@@ -826,7 +826,8 @@ Permission402=Créer/modifier les avoirs
 Permission403=Valider les avoirs
 Permission404=Supprimer les avoirs
 Permission430=Utilisez la barre de débogage
-Permission511=Lire les règlements de salaires
```
Comment (frederic34) [inline htdocs/langs/fr_FR/admin.lang:829]: @orebd you need to create en_US lang, fr_FR will be overwited by transifex process

Comment (orebd) [reply htdocs/langs/fr_FR/admin.lang:829]: Hello,
I add `Permission510=Read payments of salary` into all admin.lang
I'm right ?
I'm not sure to understand what you mean.

Comment (frederic34) [reply htdocs/langs/fr_FR/admin.lang:829]: only en_US lang is needed, other langs files are built from transifex

Comment (orebd) [reply htdocs/langs/fr_FR/admin.lang:829]: Understood.
Thank you

**Pull request closed without merging**